src/components/navbar/burgerMenu.scss

```diff
@@ -0,0 +1,52 @@
+/* Position and sizing of burger button */
+.bm-burger-button {
```

Comment:
Please use camel case for the class names, the reason for this is to being able to access these styles from JavaScript easier

Reply:
These class-names are automatically generated by `react-burger-menu`.

Reply:
then you will have to wrap them inside `:global { ..... }`, which is not ideal but needed in order to integrate libraries that are not using css modules.